diff --git a/node_modules/react-native-get-sms-android/android/build.gradle b/node_modules/react-native-get-sms-android/android/build.gradle
index 0de40fc..c2bb674 100644
--- a/node_modules/react-native-get-sms-android/android/build.gradle
+++ b/node_modules/react-native-get-sms-android/android/build.gradle
@@ -17,6 +17,7 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
+    namespace "com.react"
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
 
     defaultConfig {
